datastream-server/src/test/java/com/linkedin/datastream/server/assignment/TestStickyPartitionAssignment.java

```diff
@@ -260,12 +627,19 @@ private int getTotalPartitions(Map<String, Set<DatastreamTask>> assignment) {
   }
 
   private List<DatastreamGroup> generateDatastreams(String namePrefix, int numberOfDatastreams) {
+    return generateDatastreams(namePrefix, numberOfDatastreams, 0);
+  }
+
+  private List<DatastreamGroup> generateDatastreams(String namePrefix, int numberOfDatastreams, int minTasks) {
     List<DatastreamGroup> datastreams = new ArrayList<>();
     String type = DummyConnector.CONNECTOR_TYPE;
     for (int index = 0; index < numberOfDatastreams; index++) {
       Datastream ds = DatastreamTestUtils.createDatastream(type, namePrefix + index, "DummySource");
       ds.getMetadata().put(DatastreamMetadataConstants.OWNER_KEY, "person_" + index);
       ds.getMetadata().put(DatastreamMetadataConstants.TASK_PREFIX, DatastreamTaskImpl.getTaskPrefix(ds));
+      if (minTasks > 0) {
```

Comment:
do you see a value in validating minTasks configured to <0, I see in the code it is handled well, but this assumption in the test is different from reality and minTasks value can be set to <0.

Reply:
added a negative test, and modified this condition to be: if (minTasks != 0)